packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -116,6 +116,31 @@ const propTypes = {
    * The selected or entered date value to display in the date input.
    */
   value: PropTypes.string,
+  /**
+   * @private
```

Comment:
Why are these props private?

Reply:
These props are only internally used by DateInput component for date validation